libpkg/pkg.h.in

```diff
@@ -340,6 +340,13 @@ typedef enum {
 } mirror_t;
 
 typedef enum {
+	REPO_NONE = 0,
+	REPO_PKG,
+	REPO_DEBIAN,
+
+} type_t;
```

Comment:
That's right. I think it's a good idea then to prefix lic_t, match_t, mirror_t, signature_t too, to keep things consistent.